index.android.js

```diff
@@ -168,7 +167,6 @@ export default class LetsDoLunch extends Component {
       map = <View style = {styles.container}>
         <MapView
           style           = {styles.map}
-          region          = {region}
```

Comment:
@joel-g Isn't this needed to render the map? (I also just noticed that in the [documentation example](https://github.com/airbnb/react-native-maps/#using-a-mapview-while-controlling-the-region-as-state) they have this as `this.state.region`).

Reply:
Ok I was JUST looking at that and don't remember removing it and thought that was strange. fixing now